ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderElection.java

```diff
@@ -234,12 +234,17 @@ LifeCycle.State getCurrentState() {
 
   @Override
   public void run() {
+    if (server.initializing()) {
+        LOG.info("{}: skip running since the server is initializing", this);
```

Comment:
Update the message after changing the if-condition.

Reply:
ok, thanks your comment